examples/simple-roundtrip/README.md

```diff
@@ -33,13 +33,13 @@ Jim Bridger
 ✨  Done in 3.60s.
 ```
 
-The output "Jim Bridger" is printed after round-tripping encrypion and decryption of a Javascript
+The output "Jim Bridger" is printed after round-tripping encryption and decryption of a Javascript
 object with that value.
 
-If you look in the current directory, you'll find a *success.jpg* file. The example code encrypted
-that file to produce a *success.jpg.enc* file containing the encrypted file data, and a second file
-*success.jpt.edek* that contains the Encrypted Data Encryption Key (EDEK) that is required to
-decrypt the file. It then used that EDEK to decrypt the *.enc* file, writing a *decrypted.jpg* file.
+If you look in the current directory, you'll find a _success.jpg_ file. The example code encrypted
+that file to produce a _success.jpg.enc_ file containing the encrypted file data, and a second file
+_success.jpt.edek_ that contains the Encrypted Data Encryption Key (EDEK) that is required to
```

Comment:
I'd generally recommend not committing this many autoformatter markdown changes (that are "fixing" something not wrong). Took me a bit to see the only actual change here was a single spelling correction.

Generally less noticeable when there are more real changes at the same time.

Reply:
To me, all the changes in this file are just minor improvements. Some are formatter changes and some are typos, but they all seem to work toward the same goal of making the file a little bit better (or at least not worse, if you're indifferent about `*`/`_`). I could try to leave a comment on the PR myself calling out what "actually" changed, but to me it feels like a quick glance at the diff shows that it was all just minor edits.
Would you prefer I revert this file, or are you okay with me leaving it?

Reply:
I think neither is the right answer? Leaving a comment also isn't really a remedy. In this specific case it did happen to be just minor edits, but I had to check through all of them to see that was the case, and what had really changed. Making the file a bit better is fine (removing a single extra space, fixing a typo), but the rest doesn't make it any better or worse, it's just noise. I'm indifferent about `*`/`_` generally, but I have a preference for whatever already exists in the file so I'm not reading people's autoformatter of choice battling on a file when I look at a PR. This isn't something I'm faultless on either but it does add to the noise.